server/conf/log4j2.properties

```diff
@@ -1,4 +1,4 @@
-rootLogger.level={{ default .Env.SCALAR_DB_LOG_LEVEL "INFO" }}
+rootLogger.level=${env:SCALAR_DB_LOG_LEVEL:-INFO}
```

Comment:
Log4j2 also supports the same syntax to set configurations from environment variables.
https://logging.apache.org/log4j/2.x/manual/configuration.html#PropertySubstitution